config/s2i/jenkins/master/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openshift/jenkins-2-centos7:latest
```

Comment:
we don't need this anymore since we are pulling the OpenShift image now correct?

Reply:
You're right, fixed now